src/core/apiLayer/vacancyService.ts

```diff
@@ -0,0 +1,51 @@
+import qs from "query-string";
+
+const ServiceBaseUrl = "http://localhost:8082";
```

Comment:
use env var

Reply:
Moved to env var.

https://github.com/vonq/saleor-storefront/blob/cf8a7430cb0e64c90f44059827be4009326d849d/.env#L10